core/src/main/java/co/aikar/taskchain/TaskChain.java

```diff
@@ -35,12 +35,7 @@
 
 import co.aikar.taskchain.TaskChainTasks.*;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Why did you changed this file?

I don't like import wildcards. Maybe you can revert the changes from this file.
https://stackoverflow.com/a/147461

Reply:
Because it went against the specified code style. I updated the code in my latest commit